frontend/src/App.css

```diff
@@ -12,45 +13,27 @@ code {
     monospace;
 }
 
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
+/**Navbar**/
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.navbar_header {
+  background-color: #422c24;
   display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  justify-content: space-between;
 }
 
-.App-link {
-  color: #61dafb;
+header img {
+  width: 3rem;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.navbar_logo {
+  width: 13.6rem;
 }
 
-button {
-  font-size: calc(10px + 2vmin);
+.navbar_button {
+  background-color: transparent;
+  border-radius: 9rem;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+  margin: 2rem;
```

Comment:
margin overwrites margin-top and margin-bottom. but avoid margins on contents: prefer padding on container (navbar)